util/dvsim/SimCfg.py

```diff
@@ -126,9 +126,6 @@ def __init__(self, flow_cfg_file, proj_root, args):
 
         # Stuff below only pertains to individual cfg (not master cfg).
         if not self.is_master_cfg:
-            # Print info
```

Comment:
One possible option is to guard this with:
```python 
      if not self.select_cfgs or self.name in self.select_cfgs:
```

Reply:
Thanks Sri for the suggestion. I used your condition and applied it to all the functions behind the `log.info`. Please take a look.

Reply:
This is definitely quite confusing. Please add TODO to fix this in a better way later.

Reply:
Sounds good, will do.